tests/unit/checkboxradio/checkboxradio_core.js

```diff
@@ -0,0 +1,196 @@
+/*
+ * checkboxradio_core.js
+ */
+
+
+(function($) {
+
+module("Checkboxradio: core");
+test( "Checkbox - Initial class structure", function( assert ) {
+	expect( 4 );
+	var input = $("#check"),
+		label = $("label[for=check]");
+	ok( input.is( ":visible" ) );
+	assert.lacksClasses( label, "ui-button" );
+	input.checkboxradio();
+	assert.hasClasses( input, "ui-helper-hidden-accessible ui-checkboxradio" );
+	assert.hasClasses( label, "ui-button ui-widget ui-checkboxradio-label ui-corner-all",
+		true, "Label has proper classes" );
+});
+
+test( "Radios - Initial class structure", function( assert ) {
+	expect( 10 );
+	var inputs = $( "#radio0 input" ),
+		labels = $( "#radio0 label" );
+	ok( inputs.is( ":visible" ) );
+	labels.each( function() {
+		assert.lacksClasses( this, "ui-button" );
+	});
+	inputs.checkboxradio();
+	inputs.each( function() {
+		assert.hasClasses( this, "ui-helper-hidden-accessible" );
+	});
+	labels.each( function() {
+		assert.hasClasses( this, "ui-button" );
+	});
+});
+
+function assert(noForm, form1, form2) {
+	var others,
+		noFormElement = $( "#radio0 .ui-button" + noForm ),
+		form1Element = $( "#radio1 .ui-button" + form1 ),
+		form2Element = $( "#radio2 .ui-button" + form2 );
+
+	QUnit.assert.hasClasses( noFormElement, "ui-state-active" );
+	$( "#radio0" ).find( ".ui-button" ).not( noFormElement ).each( function(){
+		QUnit.assert.lacksClasses( this, "ui-state-active" );
```

Comment:
Never call anything via `QUnit.assert...`, pass through `assert` to access it here. Or add this helper to `QUnit.assert` as well.